### PR TITLE
Scientific computing

### PR DESCRIPTION
**Pull request by znicholls**
Add a section on scientific computing

### CODE REVIEW

File: docs/reading/good_enough_practices_in_scientific_computing_wilson_et_al.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title: "Good Enough Practices in Scientific Computing"
+author: "Greg Wilson et al."
+availability:
+  - "`Free pdf <https://arxiv.org/pdf/1609.00037v2.pdf>`_"
```
Comment (rgieseke) [inline docs/reading/good_enough_practices_in_scientific_computing_wilson_et_al.yml]: https://doi.org/10.1371/journal.pcbi.1005510
```suggestion
  - "`https://doi.org/10.1371/journal.pcbi.1005510`_"
```

Comment (rgieseke) [reply docs/reading/good_enough_practices_in_scientific_computing_wilson_et_al.yml]: Link to final paper?

File: docs/reading_list.rst [hunk 2]
```diff
@@ -26,6 +26,27 @@ Clean Code by Robert C. Martin
     - moderate
 
 
+Good Enough Practices in Scientific Computing by Greg Wilson et al.
```
Comment (rgieseke) [inline docs/reading_list.rst]: Why is this duplicated here?

File: docs/scientific_computing.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+This list of development tools is what Zeb relies on to develop scientific software reliably and reproducibly.
+Links are included with each of these tools to useful starting points.
+
+- Version control: `Git <http://swcarpentry.github.io/git-novice/>`_
+- Automating repetitive tasks: `Make <https://swcarpentry.github.io/make-novice/>`_
+- Virtual environments: `Conda virtual environments <https://medium.freecodecamp.org/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c>`_
+
+    - note the common gotcha that ``source activate`` has now changed to ``conda activate``
+    - we use conda instead of pure pip environments because they help us deal with more complicated dependencies: if you want to learn more about pip and pip virtual environments, check out `this introduction <https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/>`_
```
[the remark this answers is not in the document]
Comment (znicholls) [reply docs/scientific_computing.rst]: yes I like that piece!

File: docs/reading_list.rst (hunk 2 above)
Comment (znicholls) [reply docs/reading_list.rst]: let's discuss in #26

**Pull request merged**